packages/server/src/ui/app.css

```diff
@@ -97,6 +97,8 @@
 
   --page-sidebar-width: 260px;
   --page-sidebar-nav-item-height: 24px;
+  --page-sidebar-footer-font-size: 14px;
```

Comment:
let's not use a different font-size for this, how about `--subtext-font-size`?

Reply:
@patrickhulce I have accepted your CSS changes and I have removed --page-sidebar-footer-font-size as we don't need it any more.